applications/HDF5Application/custom_io/hdf5_file.cpp

```diff
@@ -136,15 +136,33 @@ File::File(Parameters Settings)
         m_file_id = H5Fcreate(m_file_name.c_str(), H5F_ACC_TRUNC, H5P_DEFAULT, fapl_id);
     else
     {
-        if (file_driver != "core")
-            KRATOS_ERROR_IF(H5Fis_hdf5(m_file_name.c_str()) <= 0)
-                << "Invalid HDF5 file: " << m_file_name << std::endl;
+        /* Save old error handler */
+        herr_t (*old_func)(hid_t, void*);
+        void *old_client_data;
+
+        H5Eget_auto(H5E_DEFAULT, &old_func, &old_client_data);
+
+        /* Turn off error handling */
+        H5Eset_auto(H5E_DEFAULT, NULL, NULL);
+
+        /* Probe. Likely to fail, but that’s okay */
```

Comment:
that is not a reassuring comment :sweat_smile: 

can you add a comment why this is ok?

Reply:
done :)